javascript-practice/src/js/views/userView.js

```diff
@@ -0,0 +1,20 @@
+export default class UserView {
+  constructor() {
+    this.emailData = document.getElementById('email');
```

Comment:
Consider another name, should clear what it is. For now, you get ELEMENT but name is DATA?

Btw, why define this as class variable but not another scopes?